### PR TITLE
imx93: move to use ccm_rev2 clock driver

### PR DESCRIPTION
**Pull request by CkovMk**
split from #62755

### CODE REVIEW

**Comment by danieldegrasse** [comment]
@CkovMk could you address the CI build failures please? You will also need to fix the merge conflict with this PR in order for CI to pass

**Comment by CkovMk** [comment]
Hi, @danieldegrasse 
CI issues are gone.

**Review by danieldegrasse — DISMISSED** [review]

**Comment by CkovMk** [comment]
Hi @mmahadevan108 ,
manifest updated, conflict resolved.
CI mostly passed, 1 failure is not related to this pr.
Thanks

**Review by danieldegrasse — APPROVED** [review]

**Review by JiafeiPan — APPROVED** [review]

**Comment by CkovMk** [comment]
edit commit message fix to fix issue in ci compliance check workflow.

**Review by dleach02 — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: d5e5358e56542b94bc65b6483396f50ed8f3172d
+      revision: 73620197038d7ba80fb1f9abd001828b9dd6a27e
```
Comment (JiafeiPan) [inline west.yml:196]: Change it to 78d1912dbc2b5f6e114d7fd19a31053716a3b01d?

**Review by dleach02 — CHANGES_REQUESTED** [review]
Need to synchronize some changes i the HAL

File: west.yml (hunk 1 above)
Comment (mmahadevan108) [reply west.yml:196]: @JiafeiPan , this hash seems to be the right one for this PR.

**Review by dleach02 — APPROVED** [review]

**Pull request merged**